packages/query/package.json

```diff
@@ -52,8 +52,9 @@
     "axios": "1.7.4",
     "bignumber.js": "9.1.2",
     "lodash.get": "4.4.2",
-    "p-queue": "8.0.1",
+    "p-queue": "7.4.1",
     "url-join": "5.0.0",
+    "uuid": "10.0.0",
```

Comment:
I need to use `uuid` as `crypto.randomID` seems to be not working on React Native (I tried polyfills but this is easier) 

If I don't do this queries throw the error:
```
"failureReason": [TypeError: crypto.randomUUID is not a function (it is undefined)]
```